docs/system/system-metrics.md

```diff
@@ -1466,6 +1468,77 @@ See also `MemAvailable` in [/proc/meminfo](https://man7.org/linux/man-pages/man5
 
 
 
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+### Metric: `system.linux.memory.slab.usage`
+
+<!-- semconv metric.system.linux.memory.slab.usage(metric_table) -->
```

Comment:
Could you please document [metric requirement level](https://github.com/open-telemetry/semantic-conventions/blob/c80cd1fb36f21eae0321ea1e765d7ae9d036b6d6/docs/general/metric-requirement-level.md#metric-requirement-levels)?

We usually add some form of "This metric is [required](/docs/general/metric-requirement-level.md#required)." beofre metric definition. 

It'll be very helpful when we yamlify metric requirement levels.

Reply:
Makes sense to me, added in https://github.com/open-telemetry/semantic-conventions/pull/1078/commits/bd3fd25411f68d5929a590cd294661d911110143